libs/challenge-registry/user-profile/src/lib/user-profile-starred/user-profile-starred.component.html

```diff
@@ -18,12 +18,22 @@ <h3>Starred</h3>
       </button>
     </div>
     <ng-container [ngSwitch]="activeTab">
-      <div *ngSwitchCase="tabs['challenges']" class="card-group">
-        <challenge-registry-challenge-card
-          *ngFor="let challenge of challenges"
-          [challenge]="challenge"
-        >
-        </challenge-registry-challenge-card>
+      <div *ngSwitchCase="tabs['challenges']">
+        <p-paginator
+          [first]="0"
+          [pageLinkSize]="10"
+          [rows]="limit"
+          [totalRecords]="challengeResultsCount"
+          [rowsPerPageOptions]="[10, 20, 30]"
+          (onPageChange)="onPageChange($event)"
```

Comment:
Is there a way to display the total number of results? Kind of like what PubMed and Synapse does:

**PubMed**
![Screen Shot 2022-10-03 at 11 07 56 AM](https://user-images.githubusercontent.com/9377970/193648062-931be944-cefc-464f-90d4-206b6640e3ae.png)

**Synapse**
![Screen Shot 2022-10-03 at 11 08 05 AM](https://user-images.githubusercontent.com/9377970/193648105-5562f8ff-0d9b-4c1c-aa60-f21efcaa8580.png)